lib/utils/job-utils/command-util.js

```diff
@@ -104,6 +106,63 @@ function commandUtilFactory(
         });
      };
 
+    CommandUtil.prototype.sshExec = function(cmdObj, sshSettings, sshClient) {
+        return new Promise(function(resolve, reject) {
+            if(cmdObj.timeout) {
+                setTimeout(function() {
+                    var seconds = cmdObj.timeout / 1000;
```

Comment:
Since the `seconds` is only used in the log message, Personally I don't think it's worth to do the division